docs/kubernetes/AWSIntegration.md

```diff
@@ -0,0 +1,68 @@
+# Integration with an AWS Kubernetes cluster
+This guide will walk you through how to use Calico Networking with an existing AWS-Kubernetes cluster.
+
+## Requirements
+* A working Kubernetes Deployment on AWS
+    - While any AWS deployment will do, we recommend the Kubernetes [`kube-up` guide](https://github.com/kubernetes/kubernetes/blob/release-1.0/docs/getting-started-guides/aws.md) for AWS. This guide was created with the `kube-up` script in mind.
+* SSH access to your master and minions
+    - Unless otherwise specified, the `kube-up` script will create a `kube_aws_rsa` private key in the `~/.ssh` folder which you can use to access your AWS Instances.
+    - SSH in with the following command `ssh -i </path/to/key> ubuntu@<PUBLIC_IP>`
+
+## Preparing your master services
+### Setting up Calico's etcd backend
+On your master, download our [etcd manifest](https://raw.githubusercontent.com/projectcalico/calico-kubernetes-coreos-demo/ah3-config-update/manifests/calico-etcd.manifest) and replace all instances of `<PRIVATE_IPV4>` with your master's IP or hostname. Then, install the manifest in the `/etc/kubernetes/manifests/` directory. The kubelet on your master should automatically spin up a docker container for the new etcd which can be accessed on port 6666 of your master.
```

Comment:
"...install the manifest..." makes it sound like there is an install command to run.  I'd just say "move the manifest".